SampleAPI/Controllers/EmployeeController.cs

```diff
@@ -0,0 +1,159 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SampleAPI.Constant;
+using SampleAPI.Models;
+using SampleAPI.Repository;
+
+namespace SampleAPI.Controllers
+{
+    /// <summary>
+    /// EmployeeModel controller is used to perform CRUD operations with employee entity.
+    /// </summary>
+    [Route("sampleAPI/[controller]")]
+    [ApiController]
+    public class EmployeeController : ControllerBase
+    {
+        private readonly ILogger<EmployeeController> _logger;
+        private readonly IEmployeeRepository _repository;
+        public EmployeeController(ILogger<EmployeeController> logger, IEmployeeRepository repository)
+        {
+            _logger = logger;
+            _repository = repository;
+        }
+
+        /// <summary>
+        /// Get all employee records.
+        /// </summary>
+        /// <returns>Returns the employee list.</returns>
+        [HttpGet]
+        [Route("GetAllEmployee")]
```

Comment:
Not an issue, but we discussed about having HTTP verb based routing. lets discuss this again when we can